src/screens/draft/index.tsx

```diff
@@ -104,7 +111,35 @@ const Draft = ({navigation}: TNavigation<Routes.SELECT_PLAYERS>) => {
     };
 
     dispatch(updateMatch(dataMatch));
+    if (error) {
+      console.log('Error: Dispatch CreateMatch(UPDATE)');
+      return Toast.show({
+        render: ({id}) => {
+          return (
+            <CustomToast
+              id={id}
+              description={'Cannot create match'}
+              title={'Error'}
+              status={ToastStatus.error}
+            />
+          );
+        },
+      });
+    }
```

Comment:
Is this working? We are not waiting for the response to know if we are having an error.
Also, add the error to the console log please!